### PR TITLE
[READY]Da great American foods update. Many many hamberders, resprites and more!

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request

This PR adds the following new dishes:

Cheese burger
Soylent burger
McRib
McGuffin
Chicken Sandwich
Fiesta Skewer
Boritos corn chips
Chicken meat & cutlet

This PR also resprites many old burgers such as:
Hamburger
Filet-o-Fish sandwich
Bacon burger
Big bite burger
Super big bite burger
Five alarm burger
Ghost burger

Some burger recipes have been changed to better match the sprite and ghostburger has been reworked to be much spookier.

burg_comparasion
fiesta64
ghostburgerglow64

## Why It's Good For The Game

Many of the current burger sprites are pretty sad and fast food is a fun chef gimmick.

## Changelog
:cl:
add: Several new burgers.
add: Fiesta skewer, corn chips & chicken meat.
del: some old burger sprites have been replaced.
tweak: The ghost burger is much spookier.
balance: some burger recipes have been tweaked.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
he never disappoints

**Comment by tralezab** [comment]
yeah i'll heart emoji

**Comment by cacogen** [comment]
those are very nice sprites

**Comment by Krysonism** [comment]
The new super big bite sprite is not as tall as the previous one, for that i apologize, i've tried to add egg and tomatoes as a spacer but neither looks right. The super big bite is still really big due to the added girth.

if anyone can sprite a convincing spacer layer i will try to extend it.

**Comment by stylemistake** [comment]
> Fiesta skewer

Also knows as "Shashlik". 😆

**Comment by Krysonism** [comment]
Obtaining chicken meat might at first glance seem like a chore, but chicken farming is actually a really good way of obtaining lots of meat. 

I think the only reason it isn't done more often is that the synthmeat reaction is so stupidly overpowered.

The chicken meat and meat steak currently uses the default sprites, but i will look into respriting them as well.

**Comment by Krysonism** [comment]
The meat on the fiesta skewer is kind of dwarfed by the veggies, but it would have to be made kind of huge if i want to fit more meat.

**Comment by bobbahbrown** [comment]
nice burgs

**Comment by Firecage** [comment]
Can I marry you?

**Comment by PKPenguin321** [comment]
>no home run baseball burger sprite

**Comment by Krysonism** [comment]
> 
> 
> > no home run baseball burger sprite

tf is a home run baseball burger? I barely know what baseball is. 

Our fast food generally involves putting döner kebab meat on a pizza.

**Comment by necromanceranne** [comment]
You make it with a bun and baseball bat.

**Comment by TheVekter** [comment]
> > > no home run baseball burger sprite
> 
> tf is a home run baseball burger? I barely know what baseball is.
> 
> Our fast food generally involves putting döner kebab meat on a pizza.

Dough + baseball

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 1]
```diff
@@ -309,6 +309,13 @@
 	tastes = list("crab" = 1)
 	foodtype = MEAT
 
+/obj/item/reagent_containers/food/snacks/meat/slab/chicken
+	name = "chicken meat"
+	desc = "A slab of raw chicken. Remember to wash your hands!"
+	cooked_type = /obj/item/reagent_containers/food/snacks/meat/steak/chicken
+	slice_path = /obj/item/reagent_containers/food/snacks/meat/rawcutlet/chicken
+	tastes = list("chicken" = 1)
```
Comment (spookydonut) [inline code/modules/food_and_drinks/food/snacks/meat.dm]:
```suggestion
	tastes = list("salmonella" = 1)
```

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 2]
```diff
@@ -431,6 +442,11 @@
 	cooked_type = /obj/item/reagent_containers/food/snacks/meat/cutlet/penguin
 	tastes = list("beef" = 1, "cod fish" = 1)
 
+/obj/item/reagent_containers/food/snacks/meat/rawcutlet/chicken
+	name = "raw chicken cutlet"
+	cooked_type = /obj/item/reagent_containers/food/snacks/meat/cutlet/chicken
+	tastes = list("chicken" = 1)
```
Comment (spookydonut) [inline code/modules/food_and_drinks/food/snacks/meat.dm]:
```suggestion
	tastes = list("salmonella" = 1)
```

File: code/modules/food_and_drinks/food/snacks/meat.dm (hunk 1 above)
Comment (Krysonism) [reply code/modules/food_and_drinks/food/snacks/meat.dm]: Won't this taste potentially carry over to the cooked steak?

**Comment by lolman360** [comment]
these sprites are beautiful, i'm crying ketchup

File: code/modules/food_and_drinks/food/snacks/meat.dm (hunk 1 above)
Comment (spookydonut) [reply code/modules/food_and_drinks/food/snacks/meat.dm]: Oh probably

**Pull request merged**